### ISSUE TITLE
[Fork & Maintenance Request] double-tag, autoclose-html-plus, emmet

### ISSUE THREAD

**Issue by zhaolinlau** [atom-community/atom#474]
### Have you checked for existing feature requests?

- [X] Completed

### Summary

The packages to speed up the workflow of web development and improve the convenience on completing HTML tags.

### Enter your response below:

In my opinion, a text editor which supports for web development should have some HTML tag features which are able to edit both HTML start and end tag at the same time, able to autoclose HTML tag after completing a HTML start tag like <html>, and also the emmet which very useful to speed up the workflow of web development.

### Any alternatives?

1. [double-tag](https://github.com/dsandstrom/atom-double-tag) by [dsandstrom](https://github.com/dsandstrom)

2. [autoclose-html-plus](https://github.com/binaryfunt/autoclose-html-plus) by [binaryfunt](https://github.com/binaryfunt)

3. [emmet](https://github.com/emmetio/emmet-atom) by [emmetio](https://github.com/emmetio)

**Comment by ThatXliner** [comment]
In my opinion, my editor should be minimal starting with only syntax highlighting for popular languages.

I remember there was a discussion about this somewhere.

**Comment by zhaolinlau** [comment]
> In my opinion, my editor should be minimal starting with only syntax highlighting for popular languages.
> 
> I remember there was a discussion about this somewhere.

I know, that's why it's not about feature request, it's about to pickup the packages due to atom sunsetting.

**Comment by rjousse18** [comment]
I agree about autoclose html tag, I think it can be a good native feature, but double tag or emmet should be optional

**Comment by zhaolinlau** [comment]
> I agree about autoclose html tag, I think it can be a good native feature, but double tag or emmet should be optional

Bro this is not feature request lol, this is about to keep the packages alive

**Comment by rjousse18** [comment]
Ho ok sorry, mb
